numpy/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+# very simple, just enough to start running tests
+
+class ndarray: pass
+
+class dtype: pass
+
+def array(
+    object : object,
```

Comment:
nit: PEP8 suggests not using a space before `:`

Reply:
Oh, it does? Shoot, I've been doing that wrong everywhere in my own code.

Reply:
See "Function annotations" under http://legacy.python.org/dev/peps/pep-0008/#other-recommendations